compatibility-tests/build.gradle

```diff
@@ -8,8 +8,8 @@ repositories {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_6
-    targetCompatibility = JavaVersion.VERSION_1_6
+    sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
Same here.

Reply:
I will revert it back to 1.6.

I figured out it was just my IDE internal gradle plugin.
When using gradle directly or `./gradlew` it works just fine.